Makefile

```diff
@@ -84,13 +84,15 @@ undeploy:
 
 ##################################
 
+# TODO: change the namespace and guess how to parametrize the name
```

Comment:
What exactly does this mean?

Reply:
heh, I forgot to delete that comment, good catch